aws/resource_aws_redshift_cluster_test.go

```diff
@@ -126,6 +126,32 @@ func TestAccAWSRedshiftCluster_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSRedshiftCluster_dnsName(t *testing.T) {
```

Comment:
When adding a `Computed: true` attribute that is always set, we can just add the singular attribute check (`resource.TestMatchResourceAttr` mentioned below) to the `_basic` test instead of a whole new acceptance test 👍

Reply:
👍